core/database_api.php

```diff
@@ -1323,3 +1323,23 @@ function db_mysql_fix_utf8( $p_string ) {
 function db_empty_result() {
 	return new ADORecordSet_empty();
 }
+
+/**
+ * Returns the maximum value of a valid integer for current driver, for the integer
+ * type that is actually used by Adodb for translating "I UNSIGNED", which is
+ * the type used in mantis schema for most numeric id fields.
+ * Adodb maps "I" as "I4". The actual types used for each driver are mapped in
+ * ADODB_DataDict objects, function ActualType.
+ * @return int
+ */
+function db_max_uint() {
+	# Safe assumption is for maximum value of 4 bytes signed integer: 2^31-1
+	# This is the limit for databases that dont support unsigned types, eg pgsl, mssql
+	$t_max = 2147483647;
+
+	if( db_is_mysql() ) {
```

Comment:
`DB_MAX_INT` makes sense to me, but i guess i can see why you'd call it `UINT` too